<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>latLonDialog</class>
 <widget class="QDialog" name="latLonDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lat Lon Settings</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="capture">
      <attribute name="title">
       <string>Capture &amp;&amp; Display</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>CRS/Projection of Captured Coordinate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="captureProjectionComboBox"/>
       </item>
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Custom CRS</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QgsProjectionSelectionWidget" name="captureProjectionSelectionWidget"/>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>WGS 84 (Latitude &amp; Longitude) Number Format</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="wgs84NumberFormatComboBox"/>
       </item>
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Other CRS Number Format</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="otherNumberFormatComboBox"/>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Coordinate Order (Not applicable to MGRS and WKT)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="coordOrderComboBox"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Coordinate Capture Delimiter (Not applicable to MGRS and WKT)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="delimComboBox"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Other Delimiter: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="otherTxt"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>DMS Second Precision: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="precisionSpinBox">
           <property name="maximum">
            <number>6</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="zoomTo">
      <attribute name="title">
       <string>Zoom to</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>'Zoom to' Input Coordinate CRS</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="zoomToProjectionComboBox"/>
       </item>
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Custom CRS</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QgsProjectionSelectionWidget" name="zoomToProjectionSelectionWidget"/>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Zoom to Coordinate Order</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="zoomToCoordOrderComboBox"/>
       </item>
       <item>
        <widget class="QCheckBox" name="persistentMarkerCheckBox">
         <property name="text">
          <string>Use Persistent Marker</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit">
         <property name="plainText">
          <string>Supported 'Zoom to' formats (See help for more details):

WGS 84 - Decimal, DMS, &amp; WKT degrees

MGRS - Only MGRS coordinates

Project &amp; Other CRS - Decimal &amp; WKT coordinates</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="externalMap">
      <attribute name="title">
       <string>External Map</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Select an External Map Provider</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="mapProviderComboBox"/>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Map Hints</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showPlacemarkCheckBox">
         <property name="text">
          <string>Show placemark</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="tristate">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <widget class="QSpinBox" name="zoomSpinBox">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>21</number>
           </property>
           <property name="value">
            <number>13</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Map Zoom Level: </string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="multizoom">
      <attribute name="title">
       <string>Multi-zoom</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>CRS of 'Add location' Input Coordinates</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="spacing">
           <number>7</number>
          </property>
          <property name="margin">
           <number>10</number>
          </property>
          <item>
           <widget class="QComboBox" name="multiZoomToProjectionComboBox"/>
          </item>
          <item>
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>Custom CRS</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QgsProjectionSelectionWidget" name="multiZoomToProjectionSelectionWidget"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Create Vector Layer Style</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Input .qml point vector style for Multi-zoom create layer</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <item row="1" column="0">
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>Default style for multi-location zoom new layers </string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="markerStyleComboBox"/>
            </item>
            <item row="0" column="0">
             <widget class="QLineEdit" name="qmlLineEdit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="qmlBrowseButton">
              <property name="text">
               <string>Browse</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Data Field Settings</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Number of extra data fields </string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="extraDataSpinBox">
            <property name="maximum">
             <number>10</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsProjectionSelectionWidget</class>
   <extends>QWidget</extends>
   <header>qgis.gui</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>latLonDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>latLonDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
